tests/components/airtouch5/test_config_flow.py

```diff
@@ -0,0 +1,61 @@
+"""Test the Airtouch 5 config flow."""
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.airtouch5.config_flow import CannotConnect
+from homeassistant.components.airtouch5.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+pytestmark = pytest.mark.usefixtures("mock_setup_entry")
+
+
+async def test_form(hass: HomeAssistant, mock_setup_entry: AsyncMock) -> None:
+    """Test we get the form."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == FlowResultType.FORM
+    assert result["errors"] is None
+
+    with patch(
+        "homeassistant.components.airtouch5.config_flow.validate_input",
```

Comment:
You shouldn't mock any function in the config flow. Instead, mock the library `Airtouch5SimpleClient`

Reply:
Tests are more realistic now